assignmentTest/splab/Gopistol/invitation/src/main/java/com/splab/invitation/domain/member/service/EmailService.java

```diff
@@ -0,0 +1,67 @@
+package com.splab.invitation.domain.member.service;
+
+import com.splab.invitation.domain.member.entity.Member;
+import com.splab.invitation.domain.member.service.dto.InviteMemberCommand;
+import com.splab.invitation.persistence.member.MemberRepository;
+import jakarta.mail.Message.RecipientType;
+import jakarta.mail.internet.InternetAddress;
+import jakarta.mail.internet.MimeMessage;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import lombok.RequiredArgsConstructor;
+import org.springframework.core.io.Resource;
+import org.springframework.core.io.ResourceLoader;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@Transactional
+@RequiredArgsConstructor
+public class EmailService {
+
+  private final MemberRepository repository;
+  private final JavaMailSender mailSender;
+  private final ResourceLoader resourceLoader;
+  private static final String prefix = "http://localhost:8080/members/joinMember?";
+  private static final String CONTENT_PLACEHOLDER = "CONTENT";
+  private static final String LINK_PLACEHOLDER = "YOUR_INVITATION_LINK";
```

Comment:
1) static 필드 -> instance 필드 순서로 적는게 좋습니다.
2) static final 필드들은 instance 필드와 구분하기 위해 대문자를 사용합니다.
3) EmailService가 불필요한 많은 정보를 알고 있습니다. `PREFIX`와`LINK_PLACEHOLDER` 모두 몰라도 됨
```suggestion
  private static final String PREFIX = "http://localhost:8080/members/joinMember?";
  private static final String CONTENT_PLACEHOLDER = "CONTENT";
  private static final String LINK_PLACEHOLDER = "YOUR_INVITATION_LINK";

  private final MemberRepository repository;
  private final JavaMailSender mailSender;
  private final ResourceLoader resourceLoader;
```